engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -142,7 +142,7 @@ <h4 translate>Content Host Properties</h4>
 
       <div class="detail">
         <span class="info-label" translate>RAM </span>
-        <span class="info-value">{{ host.facts["dmi::memory::size"] }}</span>
+        <span class="info-value">{{ host.facts["memory::memtotal"] }}</span>
```

Comment:
It looks like dmi::memory::size is 'humanized' in that it shows  4012 MB, but memory::memtotal is not.  It also looks like memtotal is in kilobytes rather than megabytes.  Would you be able to try to convert to megabytes and show the units (MB)?